views/user-profile.html

```diff
@@ -13,29 +13,207 @@ <h1 class="section-heading">User Profile</h1>
+                                            <form action="/user/profile/add-additional-email" method="post">
+                                                <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+                                                <td>
+                                                    <div>
+                                                        <input type="text" class="form-control" name="additional_email_2" required>
+                                                        <span class="bar"></span>
+                                                    </div>
+                                                </td>
+                                                <td>
+                                                    <input class="btn btn-primary btn-outline" type="submit" value="Add">
+                                                </td>
+                                            </form>
+                                        </tr>
+                                    {% endif %}
+                                </tbody>
+                            </table>
+                        </div>
+                    {% else %}
+                        <p class="info">No further additional emails may be added to this account.</p>
+                    {% endif %}
+                </div>
+                <div class="col-xs-12 col-lg-6 change-password">
+                    <legend>Change Password</legend>
+                    <form action="/user/profile/change-password" method="post">
+                        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+                        <div class="form-group">
+                            <label>Password</label>
                             <input type="password" class="form-control" name="password" required>
-                            <span class="highlight"></span>
                             <span class="bar"></span>
-                          
                         </div>
                         <div class="form-group">
-                             <label>Confirm</label>
+                            <label>Confirm</label>
                             <input type="password" class="form-control" name="confirm" required>
-                            <span class="highlight"></span>
                             <span class="bar"></span>
-                           
                         </div>
+                        {% include "includes/password-requirements.html" %}
                         <div class="form-group">
-                            <input class="btn btn-primary btn-outline" type="submit" value="Save">
+                            <input class="btn btn-primary btn-outline password-check" type="submit" value="Update Password" disabled="true">
                         </div>
                     </form>
+
+                    <div class="delete-account push">
+                        <legend>Delete Account</legend>
```

Comment:
Brilliant! It is great that now multiple emails can be combined into a single account and also delete the old accounts.